src/content/docs/style-guide/processes-procedures/embed-images.mdx

```diff
@@ -96,10 +96,10 @@ You can choose from a variety of icons to include in your docs:
 * New Relic icons (prefixed with `nr-`)
 * Logos for third-party products (prefixed with `logo-`)
 
-Here are the two places you can look to see if we have the icon you need:
+Here are the two places you can look to see if we have the icon you need. At the moment, these locations have separate icons:
 
-* [Gatsby theme](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) (these are icons available across the developer and docs sites)
-* [Docs site icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104)
+* [Gatsby theme](https://github.com/newrelic/docs-website/tree/develop/src/%40newrelic/gatsby-theme-newrelic/icons) (these are icons available across the developer and docs sites)
+* [Docs site Feather icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104)
```

Comment:
Could we explain the difference between Gatsby and Feather icons?

Reply:
@urbiz-nr , I will add an explanation of the two groups.